--- cntlm-0.92.3/configure.orig	2016-06-03 10:07:06.595562861 +0200
+++ cntlm-0.92.3/configure	2016-06-03 10:13:53.628885139 +0200
@@ -53,7 +53,7 @@ fi
 
 STAMP=configure-stamp
 CONFIG=config/config.h
-TESTS="endian strdup socklen_t gethostname"
+TESTS="strdup socklen_t gethostname"
 
 #[ -f $STAMP ] && exit 0
 touch $STAMP
@@ -77,3 +77,15 @@ for i in $TESTS; do
 	echo $rc >> $CONFIG
 	echo $RET
 done
+
+cat <<"EOF" >> $CONFIG
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#define config_endian 1
+#warning using LITTLE endian
+#else
+#define config_endian 0
+#warning using BIG endian
+#endif
+EOF
+
+
